src/components/settings/SettingsSSOTab/index.jsx

```diff
@@ -21,22 +24,143 @@ const SettingsSSOTab = ({ enterpriseId, setHasSSOConfig }) => {
   const [existingProviderData, pdError, pdIsLoading] = useExistingProviderData(enterpriseId, refreshBool);
   const [showNewSSOForm, setShowNewSSOForm] = useState(false);
   const [showNoSSOCard, setShowNoSSOCard] = useState(false);
+  const { AUTH0_SELF_SERVICE_INTEGRATION } = features;
+  const [isOpen, open, close] = useToggle(false);
+
+  const newConfigurationButtonOnClick = async () => {
+    for (let i = 0; i < existingConfigs.length; i++) {
+      Promise.all(LmsApiService.updateEnterpriseSsoOrchestrationRecord(
+        { active: false, is_removed: true },
+        existingConfigs[i].uuid,
+      )).then(() => {
```

Comment:
Make sure we're calling Promise.all() on all the calls, not each one individually.
```suggestion
      Promise.all(existingConfigs.map(config => LmsApiService.updateEnterpriseSsoOrchestrationRecord(
        { active: false, is_removed: true },
        config.uuid,
      ))).then(() => {
```